samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -247,7 +247,7 @@ select_the_feedback=Select the Feedback Components to be displayed to the studen
 selected=Selected
 selection_level_feed=Selection-level Feedback
 time_expired1=Time Expired!
-time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted.
+time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted in the next few minutes.
```

Comment:
@bjones86 Ahhh ha, I was driving, did not read it right (I know.. I shouldn't anyway but...)
Ok, so the queue will process items (spawn threads) with a 3 seconds delayed timer... Depending on how many items are in the queue, then each item will take N*3 secs to be submitted (ideally) ... Still, you are right, it is vague, although some people have complained auto-submission is not immediately visible, maybe re-word for "momentarily" would be better.

Thanks.